browser/src/map/handler/Map.Keyboard.js

```diff
@@ -370,7 +370,7 @@ L.Map.Keyboard = L.Handler.extend({
 	// any 'beforeinput', 'keypress' and 'input' events that would add
 	// printable characters. Those are handled by TextInput.js.
 	_onKeyDown: function (ev) {
-		if (this._map.uiManager.isUIBlocked() || (this._map.isReadOnlyMode() && !this.readOnlyAllowedShortcuts(ev))
+		if (this._map.uiManager.isUIBlocked()
```

Comment:
We removed this read-only filter for keyboard. What do we have now?
User cannot open any dialog using keyboard shortcut for example?

Reply:
Can you specify please? Keyboard events are filtered on the core side. We are planning to add some more checks but this should be working by now. If i missed anything, an example would work better for me.

Reply:
no, just asking if we protect user from doing something by keyboard shortcuts. If it is filtered on core side - even better (it's always possible to emit something manually on the wire).

Reply:
Ah okay thanks.